PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -238,59 +238,64 @@ Usage: PreeFreeSurferPipeline.sh [options]
   --t1samplespacing=<seconds>       T1 image sample spacing, "NONE" if not used
   --t2samplespacing=<seconds>       T2 image sample spacing, "NONE" if not used
   --unwarpdir={x,y,z,x-,y-,z-}      Readout direction of the T1w and T2w images (according to the *voxel axes)
            or={i,j,k,i-,j-,k-}      (Used with either a gradient echo field map 
-									or a spin echo field map)
+                                    or a spin echo field map)
   --gdcoeffs=<file path>            File containing gradient distortion
-									coefficients, Set to "NONE" to turn off
+                                    coefficients, Set to "NONE" to turn off
   --avgrdcmethod=<avgrdcmethod>     Averaging and readout distortion correction
-									method. See below for supported values.
+                                    method. See below for supported values.
+  --custombrain=(NONE|MASK|CUSTOM)  Whether to use a custom brain mask (MASK) the 
+                                    pre-existing custom brain images (CUSTOM)
+                                    or not (NONE)
 
-	  "${NONE_METHOD_OPT}"
-		 average any repeats with no readout distortion correction
+    "${NONE_METHOD_OPT}"
+     average any repeats with no readout distortion correction
 
-	  "${FIELDMAP_METHOD_OPT}"
-		 equivalent to "${SIEMENS_METHOD_OPT}" (see below)
-		 SiemensFieldMap is preferred. This option value is maintained for
-		 backward compatibility.
+    "${FIELDMAP_METHOD_OPT}"
+       equivalent to "${SIEMENS_METHOD_OPT}" (see below)
+       SiemensFieldMap is preferred. This option value is maintained for
+       backward compatibility.
 
-	  "${SPIN_ECHO_METHOD_OPT}"
-		 average any repeats and use Spin Echo Field Maps for readout
-		 distortion correction
+    "${SPIN_ECHO_METHOD_OPT}"
+       average any repeats and use Spin Echo Field Maps for readout
+       distortion correction
 
-	  "${GENERAL_ELECTRIC_METHOD_OPT}"
-		 average any repeats and use General Electric specific Gradient
-		 Echo Field Maps for readout distortion correction
+    "${GENERAL_ELECTRIC_METHOD_OPT}"
+       average any repeats and use General Electric specific Gradient
+       Echo Field Maps for readout distortion correction
 
-	  "${SIEMENS_METHOD_OPT}"
-		 average any repeats and use Siemens specific Gradient Echo
-		 Field Maps for readout distortion correction
+    "${SIEMENS_METHOD_OPT}"
+       average any repeats and use Siemens specific Gradient Echo
+       Field Maps for readout distortion correction
 
   --topupconfig=<file path>      Configuration file for topup or "NONE" if not
-								 used
+                 used
   --bfsigma=<value>              Bias Field Smoothing Sigma (optional)
+  --t1biascorrect=(YES|NONE)     Bias Field Correction for T1w image only (YES or NONE if not, YES default; optional)
+  --mppversion=(hcp|legacy)      Which version of MPP to use. ("hcp" is default, "legacy"; optional).
```

Comment:
Recapitulated from email: to me, "version" implies a number like "v4.1.0", I propose changing this to something like "--mpp-mode" (the other flags to this particular script show a distinct lack of hyphenation, but I'm not sure which convention is currently more used in the rest of the pipelines - I prefer hyphens at word boundaries).

Grega asked for a consensus from our side, so please reply here.

Reply:
To me, this actually feels more like something that we should control via an environment variable (e.g., `HCP_MPPMODE`) rather than a flag.  That said, I agree that "--mpp-mode" is better than "--mppversion" if we stick with the flag approach.  However, I'm not a particular fan of "legacy" as the name of one of the supported modes, which implies to me a reversion to some legacy _code_, as opposed to legacy-style _data_.  Perhaps name the modes "hcpStrict" and "legacyData" instead...?

Reply:
Using an environment variable instead would prevent needing to add the flag to a bunch of scripts.  In that case, we should probably interpret unset/empty string as equivalent to "hcp".

I'm not sure "hcpStrict" is good, because we probably don't want to imply there is a non-strict version of hcp mode.  I'll also put out "permissive" and "relaxed" as general terms that are appropriate to the situation.

Reply:
Unset parameter does default to `hcp`. 

I would not like to use this as an environment variable as it makes it less transparent in calls and complicates things when you run multiple studies of which some might and others might not conform to the standard. 

Regarding the values we could use strict or standard on one end and extended on the other.

Reply:
Would this work:
```
  [--mpp-mode=(standard|extended)]    Which version of MPP to use. "standard" (the default) requires the data and follows
                                      the steps described in Glasser et al. (2013). "extended" allows additional 
                                      functionality and working with data that does not conform to standards described in
                                      Glasser et al. (2013), e.g. missing high-resolution T2w image, missing field maps
                                      for distortion correction, etc.
```

Reply:
Parts of that are still not clear.  How about:

  [--mpp-mode=(standard|extended)]    Which variant of Minimal Preprocessing (MPP) to use. "standard" (the default) follows the processing steps described in Glasser et al. (2013) and requires 'HCP-Style' data acquistion. "extended" allows additional processing functionality and use of some acquisitions that do not conform to 'HCP-Style' expectations; e.g., missing high-resolution T2w image, missing field maps for fMRI distortion correction, etc.

Not sure if I'm a fan of ending this with "etc".  Just we just provide a complete albeit brief list?

Reply:
I agree better to use mode than version.

Reply:
I have adjusted the description. The list will match the relevant pipeline.

Reply:
I guess I wasn't expecting multiple studies to use the same setup script, but there currently isn't anything necessarily study-specific in it.